week4tasks/src/week4/reerser2/revsers.java

```diff
@@ -0,0 +1,16 @@
+package week4.reerser2;
+
+import java.util.Scanner;
+
+public class revsers {
+    public static void main(String[] args){
+         String S;
```

Comment:
the variables must start with lowercase letters and be meaningful ass there is no mean with S.